### PR TITLE
Adding Another Open Source Project - monkeytype

### LINKED ISSUE #390: ⭐FEAT: Add more Orgs/Projects to the /projects section
## TODO: 
- Look for more new OS orgs/ projects 
- Add more Orgs/Projects to the /projects section

Current projects :

### PR DESCRIPTION
**Pull request by sparshyadav**
Monkeytype is a open-source minimilist typing practice, testing website with some other features, and i've updated monkeytyping in the code.

Fixes #390

### CODE REVIEW

**Comment by sparshyadav** [comment]
Hi, I am a bit confused, did you close my pull request?

**Comment by ArslanYM** [comment]
> Hi, I am a bit confused, did you close my pull request?

No  I haven't

File: web/src/pages/Issues/ListOfOrgs/listOfOrgs.js [hunk 1]
```diff
@@ -277,21 +277,28 @@ const projectList = [
...
   {
     "name": "Next.js",
     "imageSrc": "https://avatars.githubusercontent.com/nextjs?s=200&v=4",
     "projectLink": "https://github.com/vercel/next.js",
     "description": "The React Framework made by Vercel.",
-    "tags": ["open-source", "typescript" ,"javascript" ,"nodejs" ,"Framework", "tailwindcss"]
+    "tags": ["open-source", "typescript", "javascript", "nodejs", "Framework", "tailwindcss"]
+  },
+  {
+    "name": "monkeytyping",
```
Comment (ArslanYM) [inline web/src/pages/Issues/ListOfOrgs/listOfOrgs.js:297]: The project name is `Monkeytype`, please make the required changes

**Review by ArslanYM — CHANGES_REQUESTED** [review]

File: web/src/pages/Issues/ListOfOrgs/listOfOrgs.js (hunk 1 above)
Comment (sparshyadav) [reply web/src/pages/Issues/ListOfOrgs/listOfOrgs.js:297]: Sure, on it.

**Pull request merged**